packages/instantsearch.js/src/widgets/index.ts

```diff
@@ -26,7 +26,7 @@ export { default as geoSearch } from './geo-search/geo-search';
 export { default as hierarchicalMenu } from './hierarchical-menu/hierarchical-menu';
 export { default as hits } from './hits/hits';
 export { default as hitsPerPage } from './hits-per-page/hits-per-page';
-export { default as index } from './index/index';
+export { default as index, type IndexWidget } from './index/index';
```

Comment:
Are you confident import statements mixing types and values work in target frameworks? I remember having issues in the past with this.

Reply:
sure, removed them just in case, probably some older tools don't remove them properly? in a major we can definitely be more consistent though (and probably export all types, but I'd do that in a separate PR)